.github/workflows/ci.yml

```
@@ -5,33 +5,31 @@ name: Java CI with Maven
 
     steps:
-    - uses: actions/checkout@v2
-
-    - name: Clone IDF Release From Github
-      uses: actions/checkout@v2
-      with:
-        repository: espressif/esp-idf
-        path: dependencies/idf-tools
-        submodules: 'true'
-        ref: release/v5.1
+    - uses: actions/checkout@v4
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 6 but found 4

(indentation)